docs/protocol-spec.md

```diff
@@ -26,6 +26,55 @@ HTTP Endpoints:
 
 While bindle names MAY be hierarchical, neither the `_i` nor the `_p` endpoints support listing the contents of a URI. This constraint is for both scalability and security reasons. To list available bindles, agents MUST use the `_q` endpoint if implemented. In absence of the `_q` endpoint, this specification does not support any way to list available bindles. However, implementations MAY support alternative endpoints, provided that the URI for those endpoints does not begin with the `_` character.
 
+## Bindle object
+
+Normally, a Bindle object consists of an `invoice` string (with the exact contents signed by the bindle creator) and one or more `signature` blocks:
```

Comment:
Do we need to specify what kind of data format the `invoice` string is? Should it be the same format as the requested bindle object or should it always be toml?

Reply:
After rereading the original issue, I think this is meant to always be TOML. If we choose to go this route of signing, that detail should be specified in the spec